app/presenters/sufia/admin_set_presenter.rb

```diff
@@ -1,4 +1,7 @@
 module Sufia
   class AdminSetPresenter < CollectionPresenter
+    def total_items(admin_set = solr_document)
```

Comment:
If I'm tracing this correctly (looking at `CollectionPresenter` in CC), the `AdminSetPresenter` should already have the necessary solr document as in ivar sitting behind an `attr_accessor`, so I'm not sure this method needs any args.